reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -22,16 +22,25 @@ package com.here.ort.reporter.reporters
 import ch.frankel.slf4k.*
 
 import com.here.ort.model.OrtIssue
+import com.here.ort.model.OrtResult
 import com.here.ort.model.VcsInfo
+import com.here.ort.model.config.CopyrightGarbage
 import com.here.ort.model.config.ScopeExclude
+import com.here.ort.reporter.Reporter
+import com.here.ort.reporter.ResolutionProvider
+import com.here.ort.reporter.reporters.TabularModelMapper.TabularScanRecord
 import com.here.ort.utils.isValidUrl
 import com.here.ort.utils.log
 
 import java.io.File
 import java.util.SortedMap
 
-class StaticHtmlReporter : TableReporter() {
-    override fun generateReport(tabularScanRecord: TabularScanRecord, outputDir: File): File {
+class StaticHtmlReporter : Reporter() {
+    override fun generateReport(ortResult: OrtResult, resolutionProvider: ResolutionProvider,
+                            copyrightGarbage: CopyrightGarbage, outputDir: File, postProcessingScript: String?)
```

Comment:
Put the parameters on separate lines:
https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting